Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Select.hs

```diff
@@ -220,3 +226,17 @@ decodeAddress BA.Address {..} = do
 
 isEmpty :: Maybe Text -> Bool
 isEmpty = maybe True (T.null . T.replace " " "")
+
+searchRetry :: Text -> Maybe Select.RetryType -> Id DSearchReq.SearchRequest -> UTCTime -> Flow ()
+searchRetry parentSearchId retryType searchid now = do
+  let justRetryType = fromMaybe (error "RetryType is Nothing") retryType
```

Comment:
use `fromMaybeM` (exposed from shared-kernel) instead of `formMaybe`.

Reply:
done